agreement/voteAggregator_test.go

```diff
@@ -839,3 +839,38 @@ func TestVoteAggregatorFiltersVoteNextRound(t *testing.T) {
 	require.NoError(t, err)
 	require.NoErrorf(t, res, "Votes from next round not correctly filtered")
 }
+
+func TestVoteAggregatorOldVote(t *testing.T) {
+	ledger := makeTestLedgerMaxBlocks(readOnlyGenesis100, 320)
```

Comment:
take a look on 
```
func balanceRound(r basics.Round, cparams config.ConsensusParams) basics.Round {
	return r.SubSaturate(basics.Round(2 * cparams.SeedRefreshInterval * cparams.SeedLookback))
}
```
this is how the 320 number is generated, and we should ensure that this is what we test against.
i.e. replace the 320 with the above calculation. you can use the current consensus protocol params to get the numerical values.